src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCaller.java

```diff
@@ -139,6 +139,7 @@
 @BetaFeature
 public final class HaplotypeCaller extends AssemblyRegionWalker {
 
+    //NOTE: many of these settings are referenced by HaplotypeCallerSpark
     public static final int DEFAULT_READSHARD_SIZE = NO_INTERVAL_SHARDING;
     public static final int DEFAULT_READSHARD_PADDING = 100;
     public static final int DEFAULT_MIN_ASSEMBLY_REGION_SIZE = 50;
```

Comment:
Did you not touch `HaplotypeCallerArgumentCollection` / `AssemblyBasedCallerArgumentCollection` / `StandardCallerArgumentCollection` (and nested arg collections) in this PR? That's where most of the `HaplotypeCaller` args live...

Reply:
When I did all the best practices tools I thought I'd do that with the other arg collections and then when I did the arg collections I thought I already did those with the best practices tools.  :(